lib/rules/indent.js

```diff
@@ -521,540 +393,486 @@ module.exports = {
          * @returns {boolean} True if the node is the outer IIFE
          */
         function isOuterIIFE(node) {
-            const parent = node.parent;
-            let stmt = parent.parent;
 
             /*
-             * Verify that the node is an IIEF
+             * Verify that the node is an IIFE
              */
-            if (
-                parent.type !== "CallExpression" ||
-                parent.callee !== node) {
-
+            if (!node.parent || node.parent.type !== "CallExpression" || node.parent.callee !== node) {
                 return false;
             }
 
             /*
-             * Navigate legal ancestors to determine whether this IIEF is outer
+             * Navigate legal ancestors to determine whether this IIFE is outer
```

Comment:
I know you inherited this, but any idea what "legal ancestors" means here? Can we improve this comment?